amy/static/online_country.js

```diff
@@ -1,52 +1,35 @@
-var prev_venue,
-    prev_address,
-    prev_latitude,
-    prev_longitude,
-    online_selected_before = false;
-
-var id_country = '#id_country, #id_event-country',
-    id_venue = '#id_venue, #id_event-venue',
-    id_address = '#id_address, #id_event-address',
-    id_latitude = '#id_latitude, #id_event-latitude',
-    id_longitude = '#id_longitude, #id_event-longitude';
+// disable latitude, longitude inputs if online country selected
+$(function () {
+  const id_country = "#id_country, #id_event-country";
+  const id_venue = "#id_venue, #id_event-venue";
+  const id_address = "#id_address, #id_event-address";
+  const id_latitude = "#id_latitude, #id_event-latitude";
+  const id_longitude = "#id_longitude, #id_event-longitude";
 
-if ($(id_country).val() == 'W3') {
-    online_selected_before = true;
-    $(id_venue).prop('disabled', true);
-    $(id_address).prop('disabled', true);
-    $(id_latitude).prop('disabled', true);
-    $(id_longitude).prop('disabled', true);
-}
+  if ($(id_country).val() == "W3") {
```

Comment:
I assume in this case w3 is online country? would it be possible to put this in a constant you can use.

Reply:
Improved, thanks!